seed/challenges/02-data-visualization-certification/sass.json

```diff
@@ -1,19 +1,307 @@
+        "  ",
+        "/*Only change code above this line.*/",
+        "</style>",
+        "",
+        "<div class=\"blog-post\">",
+        "  <h1>Blog Title</h1>",
+        "  <p>This is a paragraph</p>",
+        "</div>"
+      ],
+      "tests": [
+        ""
+      ],
+      "type": "waypoint",
+      "challengeType": 0
+    },
+    {
+      "id": "5690af845b3908042c47b32a",
+      "title": "Mixing the Cake",
+      "description": [
+        "A <code>mixin</code> lets you make groups of CSS declarations that you want to reuse throughout your site. Consider \"box-shadow\":",
+        "<blockquote>div{<br>&nbsp;&nbsp;-webkit-box-shadow: 0px 0px 4px #fff<br>&nbsp;&nbsp;-moz-box-shadow: 0px 0px 4px #fff<br>&nbsp;&nbsp;-ms-box-shadow: 0px 0px 4px #fff<br>&nbsp;&nbsp;box-shadow: 0px 0px 4px #fff<br>}</blockquote>",
+        "It would be really annyoing to re-write this over and over again for each element that we want to use <code>box-shadow</code> on, or having to change each value becuase we're testing different effects.",
+        "Thankfully, <code>Sass</code> has <code>mixins</code> - these are like functions for CSS. Here is how to write one:",
+        "<blockquote>@mixin box-shadow($x, $y, $blur, $c){<br>&nbsp;&nbsp;-webkit-box-shadow: $x, $y, $blur, $c<br>&nbsp;&nbsp;-moz-box-shadow: $x, $y, $blur, $c<br>&nbsp;&nbsp;-ms-box-shadow: $x, $y, $blur, $c<br>&nbsp;&nbsp;box-shadow: $x, $y, $blur, $c<br>}</blockquote>",
+        "You start by using the directive <code>@mixin</code> followed by a custom name. You can also include arguments as we have.",
+        "This way, we can use a single line: <code>box-shadow</code> and pass parameters without having to re-write all the vendor prefixes. Let's see how to use this:",
+        "<blockquote>div{<br>&nbsp;&nbsp;@include box-shadow(0px, 0px, 4px, #fff)<br>}</blockquote>",
+        "<h4>Instructions</h4>",
+        "Write your own <code>mixin</code> for <code>border-radius</code> and give it a <code>$radius</code> parameter. ",
+        "Then use <code>border-radius</code> to give <code>#awesome</code> a border radius of 10px."
+      ],
+      "challengeSeed": [
+        "<style>",
+        "  /*Add your mixin below this line.*/",
+        "  ",
+        "  /*Add your mixin above this line.*/",
+        "  #awesome{",
+        "    width: 150px;",
+        "    height: 150px;",
+        "    background-color: green;",
+        "    /*Only change code below this line.*/",
+        "",
+        "    /*Only change code above this line.*/",
+        "  }",
+        "</style>",
+        "",
+        "<div id=\"awesome\"></div>",
+        "  "
+      ],
+      "tests": [],
+      "type": "waypoint",
+      "challengeType": 0
+    },
+    {
+      "id": "5690af925b3908042c47b32b",
+      "title": "Using @if and @else.",
+      "description": [
+        "The @if directive is usefull if you want to test for a specifc case - it works just like the <code>if</code> statement in Javascript</code>",
```

Comment:
"Javascript" to "JavaScript"

And maybe put `<code>` tags around `@if`? @atjonathan